### ISSUE TITLE
Alan Wake Remastered - CUSA24653

### ISSUE THREAD

**Issue by GAMERRPCS3** [red-prig/fpps4-game-compatibility#1396]
### Game Name

Alan Wake Remastered

### Game code

CUSA24653

### Game version

01.00

### Used emulator's version

af25023

### Current status

Boots

### Error

An unhandled exception occurred at $00000001002AB15F:
EAssertionFailed: inconsistent resources not supported (srLayout.pas, line 533)
sceGnmRegisterResource:StandardMaterial.rfx PS material 2b00006
sceGnmRegisterResource:StandardMaterial.rfx VS material 2b00007
  $00000001002AB15F
sceGnmRegisterResource:StandardMaterial.rfx PS material 2b00007
  $00000001002D0D42
sceGnmRegisterResource:StandardMaterial.rfx VS material 2b00008
  $00000001002D0F4E
sceGnmRegisterResource:StandardMaterial.rfx PS material 2b00008
  $0000000100290EE9
sceGnmRegisterResource:StandardMaterial.rfx VS material 2b0000c
  $00000001002AAF77
sceGnmRegisterResource:StandardMaterial.rfx PS material 2b0000c
  $00000001002AB041
sceGnmRegisterResource:StandardMaterial.rfx VS material 2b0000d
  $000000010013AE3C
sceGnmRegisterResource:StandardMaterial.rfx PS material 2b0000d
  $000000010013B2A9
sceGnmRegisterResource:StandardMaterial.rfx VS material 2b00010
  $000000010013B4C1
sceGnmRegisterResource:StandardMaterial.rfx PS material 2b00010
  $000000010008E565
sceGnmRegisterResource:StandardMaterial.rfx VS material 2b00011
  $000000010008F4CD
sceGnmRegisterResource:StandardMaterial.rfx PS material 2b00011
  $000000010008FA40
sceGnmRegisterResource:StandardMaterial.rfx VS material 2b00014
  $000000010008B974
sceGnmRegisterResource:StandardMaterial.rfx PS material 2b00014
  $0000000100005056
sceGnmRegisterResource:StandardMaterial.rfx VS material 2b00015
  $00000001000144F1
sceGnmRegisterResource:StandardMaterial.rfx PS material 2b00015
sceGnmRegisterResource:StandardMaterial.rfx VS material 2b00016
  $00007FFD94BC7614
sceGnmRegisterResource:StandardMaterial.rfx PS material 2b00016
  $00007FFD95CC26A1

sceGnmRegisterResource:StandardMaterial.rfx VS material 2b00017
[TWin32WidgetSet.Destroy] WARNING: There are 1 TimerInfo structures left, I'll free them
sceGnmRegisterResource:StandardMaterial.rfx PS material 2b00017
An unhandled exception occurred at $000000010023A8CD:

### Description

starts, with a black window. then 3 seconds later it crashes.

### Screenshots

![1](https://user-images.githubusercontent.com/121258975/224441695-beee1ce0-d173-43bf-ab13-c0b119d8c082.png)
![2](https://user-images.githubusercontent.com/121258975/224441702-13edc8d6-5153-4499-84e0-c18264636ad9.png)
![3](https://user-images.githubusercontent.com/121258975/224441708-964fd209-38bb-4af0-bc95-eae1184a42b4.png)

**Comment by dev-channe** [comment]
Retest
Used emulator's version - (000305d)
Game code - CUSA24653
Game version - 01.00

![Capture d’écran 2023-11-14 211053](https://github.com/red-prig/fpps4-game-compatibility/assets/121258975/7c765daf-2d85-4660-b404-30fba96c774c)

**Comment by dev-channe** [comment]
@gandalfthewhite19890404

**Comment by gandalfthewhite19890404** [comment]
Instead of bat files launch from command line (cmd.exe)

**Comment by gandalfthewhite19890404** [comment]
Behavior was changed, previously it was able to get further. Can you find build when it was happened - last build with old behaviour and first build with new?

**Comment by dev-channe** [comment]
> Behavior was changed, previously it was able to get further. Can you find build when it was happened - last build with old behaviour and first build with new?

yes indeed and it's not the only one there are also others who can no longer even boot.
